modules/mauth-signer-akka-http/src/main/scala/com/mdsol/mauth/MAuthRequestSigner.scala

```diff
@@ -15,7 +15,7 @@ import scala.util.{Failure, Success, Try}
   * @param uri        The URI of the API call , (host name and port not included)
   * @param body       The body of the request in string form
   */
-case class UnsignedRequest(httpMethod: String = "GET", uri: URI, body: Option[String] = None, headers: Map[String, String] = Map.empty)
+case class UnsignedRequest(httpMethod: String = "GET", uri: URI, body: Option[String] = None, headers: Map[String, String] = Map.empty, contentType: Option[String] = None)
```

Comment:
`Content-Type` is another header, do we really need a parameter for it?

Reply:
How about passing it in a Map and later doing the same logic with difference that it will be taken from the map and set on the entity?

Reply:
Changed! Now Content-type is passed it in the headers map and removed later before the actual Akka Http call.

Reply:
Cool trick! 👍